.github/workflows/create-tag-with-release-pr.yml

```
@@ -0,0 +1,24 @@
+name: Create tag on release PR.
+on:
+  pull_request:
+    types:
+     - labeled
+
+jobs:
+  run-release-trigger:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'release')
+    env:
+      GH_REF: ${{ github.event.pull_request.head.ref }}
+    steps:
+      - run: echo "PR_TAG_REF=$GH_REF" | sed "s/release-//" >> $GITHUB_ENV
+      - name: Create tag
+        uses: actions/github-script@v5
+        with:
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "refs/tags/${{ env.PR_TAG_REF }}",
```

JS code could potentially be injected using branch names due to template interpolation:
```suggestion
              ref: `refs/tags/${process.env.PR_TAG_REF}`,
```